internal/dataplane/parser/testdata/golden/httproute-example/expression-routes-on_golden.yaml

```diff
@@ -0,0 +1,35 @@
+_format_version: "3.0"
+services:
+- connect_timeout: 60000
+  host: httproute..httproute-testing._.0
+  name: httproute..httproute-testing._.0
+  protocol: http
+  read_timeout: 60000
+  retries: 5
+  routes:
+  - expression: ((net.protocol == "http") || (net.protocol == "https")) && ((http.path
```

Comment:
The `.protocol` bits will change given the changes in #4422 ? Nothing major but this test will (or should) break if I understand this correctly

Reply:
Yes, #4422 changes would exhibit in this test failing if it was already merged.